docs/Doxyfile

```diff
@@ -284,8 +284,8 @@ TAB_SIZE = 4
 # @} or use a double escape (\\{ and \\})
 
 ALIASES = exceptions="\par Exceptions"
-ALIASES += noexcept="This function is `noexcept` and is guaranteed never to throw."
-ALIASES += no_libsemigroups_except="This function guarantees not to throw a ``LibsemigroupsException``."
+ALIASES += noexcept="This function is $(noexcept) and is guaranteed never to throw."
```

Comment:
I changed this because what you've written here doesn't work for me, the `$(noexcept)` is just nothing in the compiled doc, same for the one below. Is there are reason that you made this change?

Reply:
@Joseph-Edwards